src/main/java/com/puppycrawl/tools/checkstyle/xpath/ElementNode.java

```diff
@@ -200,65 +200,51 @@ public NodeInfo getRoot() {
      * Determines axis iteration algorithm. Throws {@code UnsupportedOperationException} in case,
      * when there is no axis iterator for given axisNumber.
      *
+     * Suppresses resource, IOResourceOpenedButNotSafelyClosed: {@link AxisIterator} implements
+     * {@link java.io.Closeable} interface, but none of the subclasses of the {@link AxisIterator}
+     * class has non-empty {@code close()} method.
```

Comment:
on this method we do not suppress anything, please remove this comment.

Reply:
done, 
I was referring to your previous comment "Please explain reason of suppression in javadoc", how should I explain this IDEA suppressions?

Reply:
I pushed a change